tests/integration/test_celery.py

```diff
@@ -71,3 +72,40 @@ def test_celery_run_simulation(celery_app,
     logging.info("run_simulation task finished")
     assert 'logfiles' not in result.keys()
     assert 'estimators' in result.keys()
+
+
+@pytest.mark.xfail(condition=platform.system() == "Windows", reason="FLUKA mock is not supported on Windows")
+@pytest.mark.usefixtures("live_server")
+def test_celery_run_simulation_for_fluka_mock(celery_app,
+                               celery_worker,
+                               payload_editor_dict_data_fluka: dict,
+                               client,
+                               add_simulator_mocks_to_path_variable,
+                               modify_tmpdir):
+    """Test run_simulation task with FLUKA mock binary"""
+    # lets make a local copy of the payload dict, so we don't modify the original one
+    payload_dict = copy.deepcopy(payload_editor_dict_data_fluka)
+    # limit the particle numbers to get faster results
+    payload_dict["input_json"]["beam"]["numberOfParticles"] = 12
+    payload_dict["ntasks"] = 2
+
+    files_dict, _ = files_dict_with_adjusted_primaries(payload_dict=payload_dict)
+
+    logging.info("Starting run_simulation task")
+
+    map_group = group([
+        run_single_simulation.s(
+            files_dict=files_dict,
+            task_id=str(i+1),
+            keep_tmp_files=True,  # lets pytest to clean up the tmp files, last 3 directories will be kept
+            sim_type='fluka'
+        ) for i in range(payload_dict["ntasks"])
+    ])
+
+    workflow = chord(map_group, merge_results.s())
+    job = workflow.delay()
+    logging.info("Waiting for run_simulation task to finish")
+    result: dict = job.wait()
+    logging.info("run_simulation task finished")
+    assert 'logfiles' not in result.keys()
+    assert 'estimators' in result.keys()
```

Comment:
is there some output in the estimators ? if yes, then lets check at least if the output is not empty